backend/src/main/java/com/huellapositiva/domain/model/valueobjects/ProposalRevisionEmail.java

```diff
@@ -26,7 +24,7 @@
 
     private final Reviser reviser;
 
-    private final Boolean hasFeedback;
```

Comment:
I never understood why hasFeedback is a field instead of method returning feedback != null

The logic around hasFeedback is extremelly meshy and probably wrong. Why do we need to have that as a field and not a method?

Reply:
Me neither understood it... If there are message(feedback), then it has implicit hasFeedback. I would remove it. 

In my opinion, which frontend does the string.length... hasFeedback does not contribute anything. Feedback should be optional (String).

Do I remove hasFeedback? Is extremelly meshy...

Reply:
I would. Just make it a method that returns feedback!=null

Reply:
It's done.